src/om_css/core.cljc

```diff
@@ -123,28 +123,28 @@
           (render [this] (dom/div #js { :className \"namespace_ClassName_p r\"  } ...))
        "
    [class body]
-   (letfn [(localize-classnames
-             ;; Replace any class names in map m with localized versions (names prefixed with $ will be mapped to root)
-             [class m]
-             (let [m (if (map-entry? m) m (.val m))
-                   subclass (if (map-entry? m) (.val m) (:class m))
-                   entry (fn [c]
-                           (let [cn (name c)]
-                             (if (str/starts-with? cn "$")
-                               (str/replace cn #"^[$]" "")
-                               `(om-css.core/local-class ~class ~cn))))
-                   subclasses (if (vector? subclass)
-                                (apply list (reduce (fn [acc c] (conj acc (entry c) " ")) ['str] subclass))
-                                (entry subclass))]
+    (letfn [(localize-classnames
+              ;; Replace any class names in map m with localized versions (names prefixed with $ will be mapped to root)
+              [class m]
+              (let [m (if (map-entry? m) m (.val m))
+                    class-value (if (map-entry? m) (.val m) (:class m))
+                    subclasses `(clojure.string/join (if (vector? ~class-value) " " "")
```

Comment:
So, I'm not sure this is going to work in fully general cases. 

You know as soon as we provide this people are going to write:

```
#js {:class [:a (when (:is-visible? props) :visible)]}
```

and your code is going to morph the keyword into a string and result in a weird compile-time error about strings not implementing IFn. 

I guess we can just document that you should not do fully general expressions inside of `:class`, but it is a little less than ideal to let people put in arbitrary code and try to transform it via a macro.

Ideas? I like what you're trying to do, but now that I see the implementation I'm worried about the errors people are going to be surprised by.